src/test/java/uk/gov/hmcts/juror/job/execution/service/SmtpServiceImplTest.java

```diff
@@ -1,120 +0,0 @@
-package uk.gov.hmcts.juror.job.execution.service;
-
-import org.apache.commons.lang3.RandomStringUtils;
-import org.apache.commons.lang3.RandomUtils;
-import org.junit.jupiter.api.BeforeEach;
-import org.junit.jupiter.api.Test;
-import org.mockito.ArgumentCaptor;
```

Comment:
We will want to keep this class for future use

Reply:
Dead code should not be in live code. If the code is required in the future, it can be retrieved from source control.